test/Deployment.js

```diff
@@ -30,6 +30,8 @@ describe("Deployment", function () {
     const oneYear = 365 * 86400;
     let veOLASSigners;
     let buOLASSigners;
+    // For testing of the balances creation for initial DAO members, create the file claimableBalances.json in the root folder,
+    // or copy it from scripts/deployment/dummyClaimableBalances.json.
     const jsonFile = "claimableBalances.json";
```

Comment:
This file is not needed for the deployment itself, but can be used as an inspiration for testing.